### DISCUSSION TITLE
Subfinder can not find anything

### DISCUSSION THREAD

**Discussion by justgu3st** [projectdiscovery/subfinder#881] [General]
While i was testing on last 2 days i have discovered my subfinder ### Version: v2.6.0 is not working properly.
I have used on Parrot/Kali  Any suggest ?
[https://ibb.co/HGgjWSx](url)

**Comment by MetzinAround** [comment]
Hi @justgu3st. Could you tell me more about how it's not working? Is the scan starting? What command are you using? thanks!

**Reply by justgu3st** [reply]
Yes, i am using a simple command
~ subfinder -d example.com
it gives error as attached screenshot

**Reply by MetzinAround** [reply]
That screenshot is giving me a 404. Could you copy and past the error as a code block here?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
